app/user_profile/routes.py

```diff
@@ -1,22 +1,35 @@
-from app import db
+from datetime import datetime, timezone
+from flask import render_template, request, session, jsonify
+from flask_login import login_required, current_user
+from app.utilities.utilities import format_date_local
 from app.user_profile import bp
 from app.models.user import User
-from datetime import datetime
-from flask import render_template
-from flask_login import login_required, current_user
+from app import db
```

Comment:
Do these have to be in a specific order or does it now matter?  Are you following some sort of order you created, like alphabetical?

Reply:
I believe the convention is import external and third party libraries first, or at least that's the way I have it in all the other routes and models. I haven't touched the user_profile route in a long time and while I was in the file editing the timezone route I re-organized the import statements to be consistent with everything else.